### ISSUE TITLE
Proofpoint Emerging Threats Rules

### ISSUE THREAD

**Issue by gyCfjSnO** [seud0nym/tch-gui-unhide#65]
What would be the chance of having the Proofpoint Emerging Threats Rules list added to firewall rules?

```https://rules.emergingthreats.net/fwrules/emerging-Block-IPs.txt```

**Comment by seud0nym** [comment]
This is a great suggestion. I have added a new tab to the Firewall called **Intrusion Protection**. In addition to the blocklist you suggested I have added 3 others:
```
https://rules.emergingthreats.net/blockrules/compromised-ips.txt
http://cinsscore.com/list/ci-badguys.txt
https://www.blocklist.de/downloads/export-ips_all.txt
```
These can be enabled or disabled independently of each other.

I have also added the capability for specific IPs to be added to a manual black list or white list.

I have just enabled it on my main router, so I will see what it does over the next day or so.
